### PR TITLE
Show a warning if no local composer repository is configured

### PR DESCRIPTION
**Pull request by nicodh**

### CODE REVIEW

File: Classes/Service/ExtensionService.php [hunk 1]
```diff
@@ -21,6 +21,13 @@
 
 class ExtensionService
 {
+    const COMPOSER_PATH_WARNING = "You are running TYPO3 in composer mode. You have to configure at
+        least one local path repository in your composer.json if you want to create an extensions with
```
Comment (liayn) [inline Classes/Service/ExtensionService.php:25]: > if you want to create an extensions with

if you want to create an extension with

File: Classes/Service/ExtensionService.php [hunk 2]
```diff
@@ -79,4 +86,14 @@ public function isComposerStoragePath(string $path): bool
         }
         return false;
     }
+
+    public function isStoragePathConfigured(): bool
+    {
+        if (Environment::isComposerMode() && count($this->resolveStoragePaths()) === 0) {
```
Comment (liayn) [inline Classes/Service/ExtensionService.php:92]: simplify to

`return !Environment::isComposerMode() || count($this->resolveStoragePaths());`

**Review by liayn — CHANGES_REQUESTED** [review]

File: Classes/Service/ExtensionService.php (hunk 2 above)
Comment (nicodh) [reply Classes/Service/ExtensionService.php]: count(arr) returns int not boolean so I used count(arr) > 0

Comment (liayn) [reply Classes/Service/ExtensionService.php]: > count(arr) returns int not boolean

Does not really matter. This conversion is done automatically anyways. 0 = false is the same for every programming language I know. ;-)

Comment (liayn) [reply Classes/Service/ExtensionService.php]: We could even skip the count as in PHP an empty array is `false` too.

**Review by liayn — APPROVED** [review]

**Pull request merged**